modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.types.cs

```diff
@@ -20,7 +20,10 @@ public void Add(IntPtr scriptPtr, Type scriptType)
             // TODO: What if this is called while unloading a load context, but after we already did cleanup in preparation for unloading?
 
             _scriptTypeMap.Add(scriptPtr, scriptType);
-            _typeScriptMap.Add(scriptType, scriptPtr);
+            
```

Comment:
Trailing whitespace here

Reply:
I was following the pattern below... 

https://github.com/ricaun/godot/blob/1f6b8a207a7455a72feb6ca0be3d5b59bce6065f/modules/mono/glue/GodotSharp/GodotSharp/Core/Bridge/ScriptManagerBridge.types.cs#L70-L72